dotCMS/src/main/java/com/dotcms/publisher/pusher/wrapper/PushContentWrapper.java

```diff
@@ -55,6 +56,14 @@ public void setTree ( List<Map<String, Object>> tree ) {
         this.tree = tree;
     }
 
+    public List<String> getCategories () {
+        return categories;
+    }
+
+    public void setCategories ( List<String> categories ) {
+        this.categories = categories;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "categories". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)